src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2280,9 +2280,11 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             {
 #if defined(TARGET_XARCH) && defined(FEATURE_HW_INTRINSICS)
                 // Check to see if it is possible to emulate the integer division
-                if (!(simdBaseType == TYP_INT &&
-                      ((simdSize == 16 && compOpportunisticallyDependsOn(InstructionSet_AVX)) ||
-                       (simdSize == 32 && compOpportunisticallyDependsOn(InstructionSet_AVX512)))))
+                if (varTypeIsLong(simdBaseType))
+                {
+                    break;
+                }
+                if (!compOpportunisticallyDependsOn(InstructionSet_SSE42))
```

Comment:
This is notably no longer needed. We've recently updated the "baseline" we'll target for .NET 11 from `x86-64-v1` (SSE2) to `x86-64-v2` (SSE4.2+POPCNT) and so there is no longer a defined `InstructionSet_SSE42` since its now considered "always available".